.github/workflows/deploy_lambda_staging.yml

```diff
@@ -1,9 +1,6 @@
 name: Deploy Lambda Function
```

Comment:
maybe we can call this `Deploy Staging Lambda Function` (similar to how you named the production one )